tests/test_2d_gradient/test_2d_gradient.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+import os
+import sys
+
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+from input_set import HomInfMedInput
+import openmoc
+
+
+class OneDGradientTestHarness(TestHarness):
+    """An eigenvalue calculation in a cube with vacuum BCs in x
+    and reflective BCs in y with 2-group cross section data."""
```

Comment:
Yes indeed! I also just noticed that I re-used the class name `OneDGradientTestHarness` rather than renaming appropriately to `TwoDGradientTestHarness`